tests/components/ring/common.py

```diff
@@ -15,4 +15,4 @@ async def setup_platform(hass, platform):
     )
     with patch("homeassistant.components.ring.PLATFORMS", [platform]):
         assert await async_setup_component(hass, DOMAIN, {})
-    await hass.async_block_till_done()
+    await hass.async_block_till_done(wait_background_tasks=True)
```

Comment:
Why is this change here

Reply:
I had a test failing after I added the unique_id migration code in a separate PR and I added it as per this PR  https://github.com/home-assistant/core/pull/112726 suggesting it should be in place for tests.  In the end the failure was something else so I could pull it out but it seems useful to have it there.